youtube_dl/extractor/common.py

```diff
@@ -211,6 +211,10 @@ class InfoExtractor(object):
                     specified in the URL.
     end_time:       Time in seconds where the reproduction should end, as
                     specified in the URL.
+    chapters:       A list of dictionaries, with the following entries:
+                        * "start_time" - The start time of the chapter in miliseconds
```

Comment:
i've seen that pbs and theplatform(i will try to check other) use miliseconds, if it's the most used unit it will be simpler use it directly instead of converting it into seconds in every extractor that has chapters.